daemon/src/lib.rs

```diff
@@ -195,7 +195,7 @@ pub struct Version {
     pub bios_set: bool,
     pub bios: Box<str>,
     pub description: Box<str>,
-    pub me_cr: bool,
+    pub me_cr: Option<bool>,
     pub me_hap: bool,
```

Comment:
Should we make `me_hap` optional as well? It was also missing from the pang12 changelog, so it probably would have had the same error.